packages/ocaml-variants/ocaml-variants.4.10.0+musl+static+flambda/opam

```diff
@@ -32,7 +33,14 @@ build: [
     "ASPP=musl-gcc -c"
     "--enable-flambda"
     "LIBS=-static"
-  ] {os = "openbsd" | os = "freebsd" | os = "macos" | os = "linux"}
+  ] {os = "openbsd" | os = "freebsd" | os = "macos" | os = "linux" & os-distribution != "alpine" }
+  [
+    "./configure"
+    "--prefix=%{prefix}%"
+    "CFLAGS=-Os"
+    "--enable-flambda"
+    "LIBS=-static"
+  ] {os = "linux" & os-distribution != "alpine" }
```

Comment:
You are checking `!= alpine` in both cases, so which configure is supposed to run for alpine then?

Reply:
Thanks, was obviously a typo, fixed.